Blizzard.IO.RabbitMQ.Builders/Rpc/BaseNetqRabbitRpcBuilder.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Blizzard.IO.Core.Rpc;
+using System.Collections.Generic;
+using Blizzard.IO.RabbitMQ.Rpc;
+using System;
+using Blizzard.IO.RabbitMQ.Entities.Rpc;
+using Microsoft.Extensions.Logging;
+using Blizzard.IO.Serialization.Rpc;
+
+namespace Blizzard.IO.RabbitMQ.Builders.Rpc
+{
+    public class BaseNetqRabbitRpcBuilder
+    {
+        private static Dictionary<RpcConnectionKey, INetqRabbitRpcConnection<Func<Type, object>>> netqRpcConnections = new Dictionary<RpcConnectionKey, INetqRabbitRpcConnection<Func<Type, object>>>();
+
+        protected ISerializer Serializer = new JsonSerializer();
+        protected string Hostname = "localhost";
+        protected string Password = "guest";
+        protected string Username = "guest";
```

Comment:
Put it near the other connection parametrs.

Reply:
f